.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,30 @@
+name: Sec-o-simple Website
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
Looks like this can't be merged. Maybe `push` is wrong?

Reply:
No I don't think so.
Should be just a branch permission bug, caused due to changing branch protection after this branch was created.